lib/services/batch/package.json

```diff
@@ -18,10 +18,10 @@
   "types": "./lib/batch.d.ts",
   "license": "MIT",
   "dependencies": {
-    "ms-rest": "^2.2.2",
-    "ms-rest-azure": "^2.3.3",
+    "ms-rest": "^2.3.2",
+    "ms-rest-azure": "^2.5.4",
     "underscore": "^1.8.3",
-    "moment": "^2.18.1",
+    "moment": "^2.21.1",
```

Comment:
@xingwu1 - this should be "^2.21.0". As per [npm](https://www.npmjs.com/package/moment) that is the latest version of npm that has been published. 2.21.1 is not yet available.